samplestore/src/main/java/com/stripe/samplestore/PaymentActivity.java

```diff
@@ -315,7 +318,7 @@ private void processPaymentIntent(@NonNull PaymentIntent paymentIntent) {
             confirmPaymentIntent(Objects.requireNonNull(paymentIntent.getId()));
             return;
         }
-
+        mPaymentSession.onCompleted();
```

Comment:
Are there states the PaymentIntent can be in where we shouldn't call onCompleted? In other words, should we only do this if PaymentIntent's status is succeeded? Also, should we move this call into `finishPayment()`?